frontend/.env.development

```diff
@@ -32,4 +32,4 @@ USE_SEARCH_MOCK_DATA=false
 NEW_RELIC_APP_NAME=
 NEW_RELIC_LICENSE_KEY=
 
-SESSION_SECRET=extraSecretSessionSecretValueSssh
+# SESSION_SECRET=extraSecretSessionSecretValueSssh
```

Comment:
It occurs to me we should update the dev docs for the auth work, created a separate ticket https://github.com/HHS/simpler-grants-gov/issues/3474